src/Backend/Core/Js/jquery/jquery.backend.js

```diff
@@ -1564,6 +1564,18 @@
       })
     },
 
+    getFieldIndexFromString: function(name) {
+      var chunks = name.split('_')
+
+      for(var chunk of chunks) {
+        // if the
```

Comment:
weird comment

Reply:
Fixed in 849597d0b49793df4c79d9e5237e81e488430ad4